packages/typespec-client-generator-core/test/methods/paged-operation.test.ts

```diff
@@ -18,12 +18,9 @@ beforeEach(async () => {
 
 it("azure paged result with encoded name", async () => {
   await runner.compileWithBuiltInService(`
-    op test(): ListTestResult;
-    #suppress "deprecated" "Keep for validation purposes."
-    @pagedResult
+    @list op test(): ListTestResult;
```

Comment:
These azure paging test cases could be removed directly. We already have paired one for core paging.